### PR TITLE
tiger team improvements, broken links

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** Some fixes requested by the tiger team, plus some broken link fixes
- **Is this PR related to a Monolith release?** No

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Auth JS fundamentals
 
 <ApiLifecycle access="ie" /><br>
 
-> **Note:** This document is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [Auth JS fundamentals](/docs/guides/archive-auth-js/main/). See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
+> **Note:** This document is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [Auth JS fundamentals](/docs/guides/archive-auth-js/main/). See [Identify your Okta solution](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/oie-verify-version.htm) to determine your Okta version.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md:7]: We should not be hardcoding Product docs URLs. They want us to use aliases.

File: packages/@okta/vuepress-site/docs/guides/sign-in-overview/main/index.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ After you have an idea of the above considerations, you can integrate Okta authe
...
...li>Redirect auth for <a href="/docs/guides/sign-into-web-app-redirect/">web apps</a>, <a href="/docs/guides/sign-into-mobile-app-redirect/">mobile apps</a>, and <a href="/docs/guides/sign-into-spa-redirect/">single-page apps</a></li><li><a href="/docs/guides/sampleapp-oie-redirectauth/android/main/">Redirect auth use cases</a></li><li><a href="/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget">Customize the hosted widget</a></li></ul> | <ul><li><a href="/docs/guides/embedded-siw/">Embedded widget fundamentals</a></li><li><a href="/docs/guides/oie-embedded-common-org-setup/android/main/">Sample app setup process</a></li><li><a href="/docs/guides/oie-embedded-widget-use-case-load/">Embedded widget use cases</a></li><li><a href="/docs/guides/custom-widget/main/#style-the-self-hosted-sign-in-widget">Customize the embedded widget</a></li></ul> | <ul><li><a href="/docs/guides/auth-js/">Auth JS fundamentals</a></li><li><a href="/docs/guides/oie-embedded-common-org-setup/android/main/">Sample app setup process</a></li><li><a href="/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/">Embedded SDK use cases</a></li><li><a href="/code/">Okta SDKs reference</a></li></ul>|
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-overview/main/index.md:40]: use the Markdown convention for links:
`<ul><li>Redirect auth for [web apps](/docs/guides/sign-into-web-app-redirect/), [mobile apps](/docs/guides/sign-into-mobile-app-redirect/), [single-page apps](/docs/guides/sign-into-spa-redirect/).</li><li>[Redirect auth use cases](/docs/guides/sampleapp-oie-redirectauth/android/main/)</li><li>[Customize the hosted widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)</li></ul>`

File: packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/nutrition.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-> **Note:** This document is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [Sign in to SPA with Auth JS](/docs/guides/archive-sign-in-to-spa-authjs/react/main/). See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
+> **Note:** This document is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [Sign in to SPA with Auth JS](/docs/guides/archive-sign-in-to-spa-authjs/react/main/). See [Identify your Okta solution](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/oie-verify-version.htm) to determine your Okta version.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/nutrition.md:1]: This alias is working, I don't think we need to hardcode the URL.

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
We should be using aliases for Product doc links.
We can use `<ul>` within the tables, but can we keep the Markdown convention for links?

File: packages/@okta/vuepress-site/docs/guides/sign-in-overview/main/index.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ After you have an idea of the above considerations, you can integrate Okta authe
...
 
 For more background on the different auth models, including basic flows and help with choosing between auth models, see [Okta authentication models — redirect vs. embedded](/docs/concepts/redirect-vs-embedded/).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-overview/main/index.md:42]:
```suggestion
For more background on the different auth models, including basic flows and help with choosing between auth models, see [Okta authentication models &mdash; redirect vs. embedded](/docs/concepts/redirect-vs-embedded/).
```

Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-in-overview/main/index.md:42]: LGTM; thanks!

File: packages/@okta/vuepress-site/docs/guides/sign-in-overview/main/index.md (hunk 2 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-in-overview/main/index.md]: Good call. I updated this in my last commit.

File: packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md (hunk 1 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md:7]: I discussed this with Brian earlier. I agree that using aliases makes sense, but the trouble is that they 404 for me. I've tested it on Firefox/Chrome/Safari on mac and Edge/Firefox on Windows, and the problem is the same everywhere. I also asked a couple of members of my team to test it out, and they reported the same.

I'd rather use the longer link that I know works for now, until we sort out what's going on, and then I'm happy to change to the alias once it is working.

File: packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/nutrition.md (hunk 3 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/nutrition.md:1]: See comment above

File: packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md:7]: All of our docs are using aliases for H.O.C links. I think there was an issue with the H.O.C site over the weekend, but it's resolved now. Are you still getting 404 on the H.O.C. site? If so, you should let them know.

Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md]: OK @vanngo-okta , links reverted back to alias in latest commit. Let me know if that looks OK. Thanks!

**Review by vanngo-okta — APPROVED** [review]
LGTM! Thanks Chris!

**Pull request merged**